cdp_scrapers/README.md

```diff
@@ -0,0 +1,125 @@
+# Notes on writing LegistarScrapers
+
+## Filtering
+
+1. Unimportant `EventMinutesItem`
+
+
+`EventMinutesItem(minutes_item=MinutesItem(name="123", description="This meeting 
+also constitutes a meeting of the City Council, provided ...", ...), ...)`
+
+`minutes_item.description` comes from Legistar EventItem["EventItemTitle"]. Legistar 
+EventItems wil often contain unimportant information, as shown above, that we 
+want to exclude from ingesting into CDP. The `IGNORED_MINUTE_ITEMS` attribute in 
+`LegistarScraper` 
+defies a `List[str]` that we want to filter out. 
+`IGNORED_MINUTE_ITEMS` is used in `filter_event_minutes()`, case-insensitively.
+
+In your LegistarScraper-derived class' `__init__()`, overwrite or append to 
+`self.IGNORED_MINUTE_ITEMS`. e.g.
+
+```
+class MyScraper(LegistarScraper):
+    def __init__(self):
+        super().__init__("my_city")
+
+        self.IGNORED_MINUTE_ITEMS[MinutesItem].append("This meeting also constitutes a meeting")
+```
+
+With that filter in place, this entire `EventMinutesItem` becomes `None`.
+
+2. Empty models
+
+Consider another example such as
+
+`EventMinutesItem(decision=None, matter=None, minutes_item=None, supporting_files=[], ...)`
+
+which means nothing. `MIN_INGESTION_KEYS` is the LegistarScraper attribute used 
+to return `None` for such "empty" models. Like `IGNORED_MINUTE_ITEMS` it defines a `List[str]`, 
+per `IngestionModel` class. Here, each string in the `List[str]` is a key in the 
+corresponding model. For example, the base `LegistarScraper` class defines
+
+```
+self.MIN_INGESTION_KEYS = {
+    ...
+    EventMinutesItem: ["matter", "minutes_item"],
+    ...
+}
+```
+
+`get_none_if_empty(self, model)` will test if any key in 
+`self.MIN_INGESTION_KEYS[model.__class__]` in `model` has a nonempty value; 
+`if model.__dict__[key]: ...`.
```

Comment:
nit, a safer way to do this would probably be `getattr()` instead of access the `__dict__`